monitoring/frontend.go

```diff
@@ -387,6 +387,16 @@ func Frontend() *Container {
 							PanelOptions:      PanelOptions().LegendFormat("{{category}}"),
 							PossibleSolutions: "none",
 						},
+						{
+							Name:              "90th_percentile_updatecheck_requests",
+							Description:       "90th percentile successful update requests",
+							Query:             `histogram_quantile(0.9, sum by (method,le) (rate(src_updatecheck_client_duration_seconds_bucket[5m])))`,
+							DataMayNotExist:   true,
+							DataMayBeNaN:      true,
+							Warning:           Alert{GreaterOrEqual: 0.10},
+							PanelOptions:      PanelOptions().LegendFormat("{{method}}").Max(0.10).Unit(Seconds),
```

Comment:
```suggestion
							PanelOptions:      PanelOptions().LegendFormat("{{method}}").Unit(Seconds),
```

you probably do not want max here - it will prevent seeing what the actual duration is if it is above 0.10s.

Reply:
In testing, this was needed to keep the scale readable. Most requests came in well under 1 second and the default Y-axis scale would not scale appropriately.

Reply:
Got it, LGTM then